pkg/nsx/services/staticroute/staticroute.go

```diff
@@ -135,11 +136,16 @@ func (service *StaticRouteService) GetUID(staticroute *model.StaticRoutes) *stri
 }
 
 func (service *StaticRouteService) DeleteStaticRoute(namespace string, uid string) error {
-	vpc := service.VPCService.ListVPCInfo(namespace)
-	if len(vpc) == 0 {
+	id := String(util.GenerateID(uid, "sr", "", ""))
```

Comment:
we don't need namespace as an argument any more.
shall we use uid passed from the argument instead of generating a new id?